cmd/kusk/internal/overlays/overlays.go

```diff
@@ -0,0 +1,173 @@
+package overlays
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/client"
+	"sigs.k8s.io/yaml"
+)
+
+const imageName = "jasmingacic/overlay-cli"
```

Comment:
Just as a note. There is no official image for this so I didn't wait on the owner and I built it myself

Reply:
Please make it a kubeshop image instead